service/db/src/main/java/gov/va/vro/service/db/FetchClaimsServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package gov.va.vro.service.db;
+
+import gov.va.vro.persistence.model.ClaimEntity;
+import gov.va.vro.persistence.repository.ClaimRepository;
+import gov.va.vro.service.db.mapper.ClaimMapper;
+import gov.va.vro.service.spi.model.SimpleClaim;
+import gov.va.vro.service.spi.services.fetchclaims.FetchClaimsService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class FetchClaimsServiceImpl implements FetchClaimsService {
+
+  private final ClaimRepository claimRepository;
+  private final ClaimMapper mapper;
+
+  @Override
+  public List<SimpleClaim> fetchClaims() {
+    List<ClaimEntity> claimList = new ArrayList<>();
```

Comment:
The list should not be initialized. It can all be in one line 
List<ClaimEntity> claimList =claimRepository.findAll();